src/EditorFeatures/Core.Wpf/Suggestions/SuggestedActionsSource.cs

```diff
@@ -290,6 +364,22 @@ static SuggestedActionSetPriority ConvertToSuggestedActionSetPriority(UnifiedSug
             {
                 this.AssertIsForeground();
 
+                return GetCodeFixesAsync(
+                    supportsFeatureService, requestedActionCategories, workspace, document, range,
+                    addOperationScope, highPriority: null, isBlocking: true, cancellationToken).WaitAndGetResult(cancellationToken);
```

Comment:
the `bool?` is effectively a tri-state.  It means:

1. `true`: i want high pri, only give me high pri.
2. `false`: i want noral pri, only give me normal pri.
3. `null`: i don't care.  give me both. (legacy behavior).

An enum would likely be better here, i just wasted sure what teh right location was to define it.

Reply:
Switched to an enum.